### PR TITLE
Task/cooks 165 update instructions

### PR DESCRIPTION
**Pull request by taoteg**
## Overview: ##

Major refactor if the way Report Instructions are displayed across all report views. 

## Related Jira tickets: ##

* [COOKS-165](https://jira.tacc.utexas.edu/browse/COOKS-165)

## Summary of Changes: ##

Centralizes all instructions in a single component that takes a single string parameter (the instructions to display) as an argument. Can be customized per report.

## Testing Steps: ##

1. Build project.
2. View any report view.
3. Check for instructions at first load per report view

## UI Photos:

Screen Shot 2021-12-01 at 12 15 51 AM

Screen Shot 2021-12-01 at 12 16 02 AM

Screen Shot 2021-12-01 at 12 16 51 AM

Screen Shot 2021-12-01 at 12 17 09 AM

Screen Shot 2021-12-01 at 12 18 24 AM

Screen Shot 2021-12-01 at 12 18 46 AM

## Notes: ##

None.

### CODE REVIEW

**Comment by taoteg** [comment]
FYI - this is deployed on PPRD right now.

File: client/src/components/ProTx/components/charts/PredictiveFeaturesPlot.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
 import './PredictiveFeaturesPlot.css';
 
 function PredictiveFeaturesPlot({
+  mapType,
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/charts/PredictiveFeaturesPlot.js:13]: is this needed?

**Review by nathanfranklin — APPROVED** [review]
LGTM .  I just left one comment to address (about a property that isn't used.)

File: client/src/components/ProTx/components/charts/PredictiveFeaturesPlot.js (hunk 1 above)
Comment (taoteg) [reply client/src/components/ProTx/components/charts/PredictiveFeaturesPlot.js:13]: It isn't used, but the same goes for ObservedFeaturesPlot. I'll make another PR to remove all the unused props that I ended up with after refactoring the instructions component.

**Pull request merged**